Portal/View/Component.swift

```diff
@@ -141,8 +141,8 @@ extension Component {
         case .mapView(let properties, let style, let layout):
             return .mapView(properties, style, layout)
 
-        case .imageView(let image, let style, let layout):
-            return .imageView(image, style, layout)
+        case .imageView(let image, let clipToBounds, let style, let layout):
```

Comment:
Why are you adding this in this PR? It does not seem related to adding shadows? Also components should not have more than 3 associated values. Please sent a new PR adding this property as part of a `ImageProperties` struct which contains both `image` and `cliptToBounds` properties.